mixcoatl/resource.py

```diff
@@ -8,7 +8,89 @@
 from mixcoatl.settings.load_settings import settings
 from mixcoatl.decorators.lazy import lazy_property
 from mixcoatl.utils import camel_keys
+import json
 
+class Endpoint(object):
+    """ A class for representing DCM endpoints and loading them from json files
+    """
+
+    def __init__(self, url = None, basepath=None, api_version=None, secret_key=None, access_key=None, ssl_verify=True, nickname=None):
```

Comment:
nit: delete the spaces: url=None (sorry, I am not happy that I was compelled to make this comment).